utils_cv/tracking/model.py

```diff
@@ -177,15 +178,51 @@ def fit(
         trainer.set_device(
             self.opt.gpus, self.opt.chunk_sizes, self.opt.device
         )
-
+        
+        # initialize loss vars
+        self.losses_dict = defaultdict(list)
+        self.epochs = []
+        
         # training loop
         for epoch in range(self.start_epoch + 1, self.opt.num_epochs + 1):
+            print("Training in epoch ", epoch,"...")
             mark = epoch if self.opt.save_all else "last"
             log_dict_train, _ = trainer.train(epoch, train_loader)
             if epoch in self.opt.lr_step:
                 lr = self.opt.lr * (0.1 ** (self.opt.lr_step.index(epoch) + 1))
                 for param_group in optimizer.param_groups:
                     param_group["lr"] = lr
+                    
+            # store losses in each epoch            
+            self.epochs.append(epoch)
+            for k, v in log_dict_train.items():
+                if k in ['loss', 'hm_loss', 'wh_loss', 'off_loss', 'id_loss']:
+                    self.losses_dict[k].append(v)
+
+    def plot_training_losses(self, figsize: Tuple[int, int] = (10, 5))->None: 
```

Comment:
Please move this function to plot.py

Reply:
FWIW in the detection scenario `plot_precision_loss_curve` is in the model class

Reply:
@PatrickBue  in the OD scenario,  `plot_precision_loss_curve` is in the model class. Should I still move this function to plot.py?

Reply:
If I move `plot_training_losses` to plot.py, should I change `fit` to return `self.losses_dict` , and pass it on in the notebook with, i.e. `losses_dict = fit(...)` and `plot_training_losses(losses_dict)` ?

Reply:
Good point, for object detection the plot_ function is in models.py: https://github.com/microsoft/computervision-recipes/blob/master/utils_cv/detection/model.py#L549
In that case this function can stay here. My comments about self.epochs stays the same, I don't think it's needed.